Yoast/wordpress-seo
# CS: Output escaping

## Context


* Handles four files from the CS Cleanup sheet

## Summary


This PR can be summarized in the following changelog entry:

* Fixes output escaping in `admin/views/tabs/metas/paper-content/post_type/woocommerce-shop-page.php`.
* Removes deprecated file: `admin/views/form/fieldset.php`.
* Ignores output escaping in `src/integrations/admin/migration-error-integration.php`
* Ignores output escaping in `src/integrations/front-end/force-rewrite-title.php`

## Relevant technical choices:

* Instead of fixing the output escaping in the fieldset, we removed it because it was already deprecated since version 11.9.

## Test instructions

### Test instructions for the acceptance test before the PR gets merged
This PR can be acceptance tested by following these steps:

* Travis should have less errors.
* You could run `composer cs 4` on this branch.
* Activate WooCommerce and our WooCommerce plugin.
* Visit the settings page: SEO -> Search appearance -> Content types -> Products -> Settings for Products archive
* Verify that the strings are properly translated. These are the ones that are now escaped.
  * `Settings for Products archive`
  * `You can edit the SEO meta-data for this custom type on the Shop page.`
* Verify the link works (and goes to your shop page).


### Test instructions for QA when the code is in the RC




QA can test this PR by following these steps:

* Activate WooCommerce and our WooCommerce plugin.
* Visit the settings page: SEO -> Search appearance -> Content types -> Products -> Settings for Products archive
* Verify that the strings are properly translated. These are the ones that are now escaped.
  * `Settings for Products archive`
  * `You can edit the SEO meta-data for this custom type on the Shop page.`
* Verify the link works (and goes to your shop page).

## Impact check

This PR affects the following parts of the plugin, which may require extra testing:

* Search for the removed `fieldset.php` file? It should not be used anywhere.

## UI changes


## Documentation


## Quality assurance


Fixes #
